### DISCUSSION TITLE
GUI is Too Small / Hard To See The Knobs

### DISCUSSION THREAD

**Discussion by Nick3Nick** [apohl79/audiogridder#1117] [Help] [answered]
When I open a plugin to make a change, the GUI is too small and hard to see the finer details and make accurate moves. Image attached.

![GUI Too Small Screenshot](https://user-images.githubusercontent.com/120556575/208284326-ec2c46ac-48c2-4b83-885b-ade0e387bfa9.jpg)

• AG 1.1.1
• Zoom is set to 100%
• AMD Ryzen 7 5800H with Radeon Graphics 3.20 GHz
• Windows 11

**Comment by oldtos** [comment]
Upper Left just right of server name there is graphic that looks like a mixer...click for drop down menu... select user interface..select zoom.

**Reply by Nick3Nick** [reply]
Zoom is set to 100%. Seems like there's something wrong it it's at 100% and still too small?

**Comment by rncwalker** [comment]
The size also relies on the VST it self.
In Audio Gridder change it to 150%

Should look better.

Robert :)

**Reply by Nick3Nick** [reply]
It looks better but I seem to be losing stability when trying to open the GUI

**Comment by Nick3Nick** [comment]
Changing the zoom isn't working well. In the image I've attached, you can see the top of the plugin is cut off. When I start to mess around with the viewing options, I eventually can't make changes at all and the cursor on the main machine doesn't track the same as on the server machine.

**Comment by Nick3Nick** [comment, marked as answer]
The server was running on windows 8.1. I've had resolution problems before with 8.1.

Just updated to windows 10 on the server PC and the scaling is correct now everything is running even more stable.

The problems seem to have been fixed by upgrading to Windows 10. It seems like windows 8.1 was the problem.
